### DISCUSSION TITLE
12Pピンヘッダの差し込みについて

### DISCUSSION THREAD

**Discussion by relaxman86** [Yowkees/keyball#269] [Q&A]
初めて自作キーボードに挑戦している初心者です。
ビルドガイドに沿って作業していたのですが、
12Pピンソケットをはんだ付けで固定する際に、穴にはんだが流れ込んだ影響？で
ピンヘッダを差し込んだのですが、写真のように高さがまちまちになってしまっています。

このまま作業進めていいものか悩んでおり、アドバイスいただけると助かります。
よろしくお願いいたします。
![20230704_182619](https://github.com/Yowkees/keyball/assets/138578615/f866d85a-1651-4938-8af4-c949db2dbca8)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ProMicroに同梱の12Pピンヘッダーを使用してしまったのですね。
この場合、ピンが長いものはニッパでカットして高さをそろえると完成できると思います。

ただKeyballキットに同梱の12Pピンヘッダーを使用していただくのが正規の方法にはなりますので、交換していただくのがベストです。
お手数ですがご検討お願いします。

**Comment by relaxman86** [comment]
回答ありがとうございます。いえ、キット同梱のピンヘッダーですが、ピンソケットをはんだ付けする際にはんだを流しすぎて穴が浅くなってしまったのが原因です。ピンソケットを新たに調達し、再挑戦して何とか完成しました。キーが2か所反応しないですが。。。
申し訳ないですが、また質問させていただきます。
